impact.md

```diff
@@ -49,10 +49,10 @@ receive $70,000 from the Brown Institute for Media Innovation Magic Grant.
 
 ## <a name="placements"></a>Placements
 
-> *Aaron Plasek* admitted Columbia History Ph.D. program (Spring 2015).
+> *Aaron Plasek* admitted to the Columbia History Ph.D. program (Spring 2015).
```

Comment:
good catch---separate into a separate PR pls (they should be grouped semantically, otherwise it becomes difficult to revert)